app/controllers/application_controller.rb

```diff
@@ -6,7 +6,7 @@ class ApplicationController < ActionController::API
 
   def authenticate_request
 
-    unless request['action'] == 'create' && request['controller'] == 'user'
+    unless request['action'] == 'create' || request['action'] == 'forgot' || request['action'] == 'reset' && request['controller'] == 'user' || request['controller'] == 'password'
```

Comment:
As this becomes bigger, you can use include? for cleaner code https://apidock.com/ruby/Array/include%3F